javascript/javascript1/week3/array_methods.js

```diff
@@ -0,0 +1,13 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+names.splice(names.indexOf(nameToRemove), 1); //removes one element at specified index
```

Comment:
Just learned something new. If "Ahmad" wasn't in the array, or there was just a new name used for `nameToRemove` then the index would be `-1` and it will basically remove the last item from the array. This certainly would be a mistake. To prevent it, I'd suggest a small check as this:

```
const indexOfNameToRemove = names.indexOf(nameToRemove);

if (indexOfNameToRemove >= 0) {
  names.splice(indexOfNameToRemove, 1);
}
```

Reply:
Nice! Good to know :)